### PR TITLE
In ac-dcd-find-imports-std use insert-file-contents instead of find-f…

### PR DESCRIPTION
**Pull request by nordlow**
…ile to avoid find-file-hooks being called thus speeding up things if these hooks are slow.

### CODE REVIEW

File: ac-dcd.el [hunk 1]
```diff
@@ -537,8 +537,8 @@ dcd-client outputs candidates that begin with \"this\" when completing struct co
     (with-current-buffer buf
       (erase-buffer)
 
-          (apply 'call-process-region (point-min) (point-max)
```
Comment (atilaneves) [inline ac-dcd.el:540]: ?

**Comment by atilaneves** [comment]
I'm going to just trust you on this.

File: ac-dcd.el (hunk 1 above)
Comment (nordlow) [reply ac-dcd.el:540]: Correction of indentation that sneaked in without me noticing it.

**Comment by nordlow** [comment]
(with-temp-buffer
   (insert-file-contents SOME-FILENAME)
   (goto-char (point-min))
   ;; process...
   )

is the way to parse contents of files in Lisp.

It avoids running find-file-hooks that on my setup is equals to

`(yas-global-mode-check-buffers global-flycheck-mode-check-buffers gdb-find-file-hook generic-mode-find-file-hook global-subword-mode-check-buffers undohist-recover-safe global-hl-todo-mode-check-buffers projectile-find-file-hook-function auto-revert--global-adopt-current-buffer auto-revert-find-file-function make-buffer-read-only-auto-reverted-if-uneditable mode-local-post-major-mode-change url-handlers-set-buffer-mode global-eldoc-mode-check-buffers global-font-lock-mode-check-buffers epa-file-find-file-hook vc-refresh-state c-auto-insert-empty-file-template display-main-function-eval-tip)`

:)

**Pull request merged**